drivers/pds/lib/utils.go

```diff
@@ -1441,6 +1441,30 @@ func RegisterClusterToControlPlane(infraParams *Parameter, tenantId string, inst
 	return nil
 }
 
+// Check if a deployment specific PV and associated PVC is still present. If yes then delete both of them
+func DeletePvandPVCs(resourceName string) error {
+	log.Debugf("Starting to delete the PV and PVCs for resource %v\n", resourceName)
+	pv_list, err := k8sCore.GetPersistentVolumes()
+	if err != nil {
+		if strings.Contains(err.Error(), "not found") {
+			return fmt.Errorf("persistant volumes Not Found due to : %v", err)
+		}
+		return nil
+	}
+	for _, vol := range pv_list.Items {
+		claimName := vol.Spec.ClaimRef.Name
+		if strings.Contains(claimName, resourceName) {
+			log.Debugf("strings.Contains : %v\n", strings.Contains(claimName, resourceName))
```

Comment:
can you modify this as log.Debugf("claim :%s identified",claimName)

Reply:
Done!